apps/web/src/app/app/(dashboard)/[workspaceSlug]/monitors/[id]/data/page.tsx

```diff
@@ -28,12 +28,12 @@ export default async function Page({
   params: { workspaceId: string; id: string };
   searchParams: { [key: string]: string | string[] | undefined };
 }) {
-  const workspaceId = Number(params.workspaceId);
-  const id = Number(params.id);
+  const workspaceId = params.workspaceId;
```

Comment:
Be careful, you have renamed it to: `workspaceSlug`. Check it the different places.